doc/users/whats_new.rst

```diff
@@ -138,44 +141,50 @@ rectangle for the size bar.
 Annotation can use a default arrow style
 ----------------------------------------
 
-Annotations now use the default arrow style when setting `arrowprops={}`,
+Annotations now use the default arrow style when setting ``arrowprops={}``,
 rather than no arrow (the new behavior actually matches the documentation).
 
 
 Orthographic projection for mplot3d
 -----------------------------------
-:class:`~mpl_toolkits.mplot3d.axes3d.Axes3D` now accepts ``proj_type`` kwarg and has a method :meth:`~mpl_toolkits.mplot3d.axes3d.Axes3D.set_proj_type`. The default option is ``'persp'`` as before, and supplying ``'ortho'`` enables orthographic view.
+:class:`~mpl_toolkits.mplot3d.axes3d.Axes3D` now accepts ``proj_type`` keyword
+argument and has a method :meth:`~mpl_toolkits.mplot3d.axes3d.Axes3D.set_proj_type`.
+The default option is ``'persp'`` as before, and supplying ``'ortho'`` enables
+orthographic view.
 
 
 ``voxels`` function for mplot3d
 -------------------------------
-:class:`~mpl_toolkits.mplot3d.axes3d.Axes3D` now has a ``voxels`` method, for
-visualizing boolean 3d data. Uses could include plotting a sparse 3D heat map,
-or visualizing a volumetric model.
+:class:`~mpl_toolkits.mplot3d.axes3d.Axes3D` now has a
+`~mpl_toolkits.mplot3d.axes3d.Axes3D.voxels` method, for visualizing boolean 3d
+data. Uses could include plotting a sparse 3D heat map, or visualizing a
+volumetric model.
 
 
 Barbs and Quiver Support Dates
 ------------------------------
 
-When using the :func:`quiver` and :func:`barbs` plotting methods,
-it is now possible to pass dates, just like for other methods like
-:func:`plot`. This also allows these functions to handle values
-that need unit-conversion applied.
+When using the :func:`~matplotlib.axes.Axes.quiver` and
+:func:`~matplotlib.axes.Axes.barbs` plotting methods, it is now possible to
+pass dates, just like for other methods like :func:`~matplotlib.axes.Axes.plot`.
+This also allows these functions to handle values that need unit-conversion
+applied.
 
 
 Hexbin default line color
 -------------------------
 
-The default ``linecolor`` kwarg for :func:`hexbin` is now ``'face'``, and
-supplying ``'none'`` now prevents lines from being drawn around the hexagons.
+The default ``linecolor`` keyword argument for :func:`~matplotlib.axes.Axes.hexbin`
+is now ``'face'``, and supplying ``'none'`` now prevents lines from being drawn
+around the hexagons.
 
 
-figure.legend() can be called without arguments
+Figure.legend() can be called without arguments
 -----------------------------------------------
 
-Calling :func:`figure.legend` can now be done with no arguments. In this case a
-legend will be created that contains all the artists on all the axes contained
-within the figure.
+Calling ``Figure.``:func:`~matplotlib.figure.Figure.legend` can now be
```

Comment:
Does
```
`.Figure.legend`
```
or
```
:meth:`.Figure.legend`
```
work?

Reply:
Ah, I wasn't sure if we could use the leading dot, but it does work in this case.